github/ci/prow/files/labels.yaml

```diff
@@ -2,9 +2,9 @@
 default:
   labels:
     - color: 8452c9
-      name: triage/build-officer
+      name: triage/build-watcher
       target: issues
-      description: Discovered by a kubevirt build-officer. Pay attention to these issues to keep CI healthy.
```

Comment:
According to [documentation](https://github.com/kubernetes/test-infra/tree/master/label_sync#configuration) you should add a `previously:` section, as you are modifying an existing label.

Reply:
also for the description ?
because we do have previously

(see also https://github.com/kubevirt/project-infra/pull/626#discussion_r484357055)

Reply:
The `previously` was there from a previous label-sync run. IIUC we now need to update that section from

```
previously:
   - name: build-officer
      ...
```

to 

```
previously:
   - name: triage/build-officer
      ...
```

as the label currently is `triage/build-officer`.

Reply:
Sorry for marking the wrong line BTW.

Reply:
on it, 
we need only triage/build-officer then ?
no need to cascade both right ? (in case there is an option to even)

Reply:
Done thanks

Reply:
it would have been an array, you normally put all previous labels in that array. But no old mentions are left, so should not be an issue.

Reply:
Thanks both of you